README.md

```diff
@@ -1,15 +1,17 @@
-# Aries Framework Rust
+# <img alt="Hyperledger Aries logo" src="docs/aries-logo.png" width="45px" /> Aries Framework Rust
 
-![CI build](https://github.com/AbsaOSS/libvcx/workflows/CI/badge.svg)
-[![codecov](https://codecov.io/gh/AbsaOSS/libvcx/branch/master/graph/badge.svg)](https://codecov.io/gh/AbsaOSS/libvcx)
+![CI build](https://github.com/hyperledger/aries-framework-rs/workflows/CI/badge.svg)
+[![codecov](https://codecov.io/gh/hyperledger/aries-framework-rs/branch/master/graph/badge.svg)](https://codecov.io/gh/hyperledger/aries-framework-rs)
 
-- Aries c-callable implementation written in Rust with language wrappers available in:
+Join our [Hyperledger RocketChat channel](https://chat.hyperledger.org/channel/aries-framework-rust)
```

Comment:
We are quite inconsistent in naming of the library. I can easily someone being confused if there is a difference between "aries-framework-rust" and "aries-framework-rs"

Reply:
@ryjones  is it possible to change the name of rocketchat channel `aries-framework-rust` to `aries-framework-rs` for consistency?

Reply:
Yes